app/views/content/is-teaching-right-for-me/_get_school_experience.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="campaign">
+  <article>
+    <section class="grey">
+      <section class="container">
+        <%= render(CallsToAction::Promo::PromoComponent.new(border: :none, reverse: true)) do |promo| %>
+          <% promo.left_section(heading: nil, classes: %w[history-background]) %>
```

Comment:
@ethax-ross for some reason, even though it says 'history-background', the image that's coming up for me is the events image? But we actually already use the history image further up in another section, so should probably use a different one here anyway - could we add another? Maybe one of the hero images - 4 or 21?